.travis.yml

```diff
@@ -84,12 +88,3 @@ script:
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then wget https://scrutinizer-ci.com/ocular.phar; fi
   - if [[ "$PHP_COVERAGE" = "TRUE" ]]; then php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi
 
-matrix:
```

Comment:
Moreover we use the matrix to test against both php5.6 and php7.0 as well as pgsql, sqlite and mysql.